### PR TITLE
Update license copyright to Google Inc

### PR DESCRIPTION
**Pull request by alexsinger**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context
Google suggested the change now that fastlane is part of Google Inc

### Description
Associated the fastlane copyright to Google Inc

### CODE REVIEW

**Comment by KrauseFx** [comment]
I'll do some research if that's okay to do, as we'd change the license of code that was contributed before the Google CLA was introduced.

**Review by taquitos — APPROVED** [review]
~🐑 it~ Sorry, just saw your comment Felix!

**Review by hjanuschka — CHANGES_REQUESTED** [review]
can you clarify what that means  for all pre-CLA commits - to every single contributor?

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Pull Request is Work in Progress</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by alexsinger** [comment]
@hjanuschka we asked for clarification on this from the OSS team at Google. This change doesn't affect contributor's ownership over their contributions.

**Comment by mfurtak** [comment]
nit: does legal require the capital "F" in _fastlane_? 😄

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Felix Krause
+Copyright (c) 2015-present the fastlane Authors
```
Comment (KrauseFx) [inline LICENSE]: Shouldn't authors also be lower case?

Comment (alexsinger) [inline LICENSE]: Was following legal's spelling "the Fastlane Authors", but then lower cased "fastlane" for branding reasons.

**Review by mfurtak — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (KrauseFx) [inline LICENSE]: I'd probably go with `authors`

**Review by hjanuschka — APPROVED** [review]

**Pull request merged**